toolkits/global/packages/global-lists/scss/10-settings/_default.scss

```diff
@@ -0,0 +1,17 @@
+$global-list-columned--column-gap: spacing(24);
+$global-list-columned--margin-bottom: 0.25em;
+
+$global-list-description--wrapper-margin-bottom: spacing(24);
+$global-list-description--font-weight: $font-weight-bold;
+$global-list-description--font-size: $font-size-sm;
+$global-list-description--line-height: 1.4;
+$global-list-description--margin-bottom: 0.25em;
+$global-list-description--is-interface-link: true;
+
+$global-list-group--border-color: greyscale(80);
+$global-list-group--padding-xs: spacing(4) 0;
+$global-list-group--padding-sm: spacing(8) 0;
+$global-list-group--padding-md: spacing(16) 0;
+$global-list-group--padding-lg: spacing(24) 0;
+$global-list-group--line-height: 1.4;
+$global-list-group--is-interface-link: true;
```

Comment:
Are these the styles from the springer brand? We need to have `10-settings/_springer.scss` as well for them to migrate to, otherwise if that brand needs to change then they will need to change default which will affect all the other brands.

Make the `default` really generic, don't use mixins but use explicit values e.g. `$global-list-group--border-color: #eee;`

Reply:
Gotcha, will do thank you